servers/republik/modules/crowdfundings/graphql/resolvers/Membership.js

```diff
@@ -126,11 +127,15 @@ module.exports = {
     return pgdb.public.membershipCancellations.find({
       membershipId: id
     })
-      .then(cancellations => cancellations.map(cancellation => ({
-        ...cancellation,
-        category: {
-          type: cancellation.category
-        }
-      })))
+      .then(cancellations => cancellations
+        .map(cancellation => ({
+          ...cancellation,
+          category: {
+            type: cancellation.category
+          }
+        })
+        )
+        .sort((a, b) => ascending(a.createdAt, b.createdAt))
```

Comment:
Would sort in database query wrapper with `orderBy`. It is fair to assume Postgres is faster in sorting.